airbyte-integrations/connectors/destination-postgres/build.gradle

```diff
@@ -28,8 +28,21 @@ application {
 
 dependencies {
     implementation libs.postgresql
+    // TODO: declare typing-deduping as a CDK feature instead of importing from source.
+    implementation project(':airbyte-cdk:java:airbyte-cdk:typing-deduping')
 
     testImplementation libs.testcontainers.postgresql
 
     integrationTestJavaImplementation libs.testcontainers.postgresql
+
+    integrationTestJavaImplementation testFixtures(project(':airbyte-cdk:java:airbyte-cdk:typing-deduping'))
+    //TODO: Needs dependency on source-postgres testFixtures for PostgresContainerFactory & PostgresDatabase.
+    // Move it to common module in cdk ?
+    integrationTestJavaImplementation testFixtures(project(':airbyte-integrations:connectors:source-postgres'))
```

Comment:
please don't make a destination connector depend on a source connector. There's already way too much coupling in our codebase

Reply:
@stephane-airbyte  Thanks for taking a look, I was gonna ask about this today 😅   I'm not too keen on this too but I might have to end up copy pasting the 2 classes that exist in testFixtures into destinations. Do you have any suggestions to pull into common module somwhere ?

Reply:
Removed dependency and copied the classes. Will revert source changes